src/utils/valid-vitest-globals.ts

```diff
@@ -0,0 +1,11 @@
+export const VITEST_GLOBALS = new Set([
+  'describe',
```

Comment:
we can extend this set to include more elements like `vi`,`assert`,`expectTypeOf ` and more. double check vitest docs.